hedera-node/src/test/java/com/hedera/services/sigs/order/HederaSigningOrderTest.java

```diff
@@ -1843,7 +1843,7 @@ private SigMetadataLookup hcsMetadataLookup(JKey adminKey, JKey submitKey) {
 		);
 	}
 
-	private List<Key> sanityRestored(List<JKey> jKeys) {
+	public static List<Key> sanityRestored(List<JKey> jKeys) {
```

Comment:
Seems like its scope was extended for local testing in `JKeyTest`. Should revert back to `private` and remove unused import's in the test.